discretisedfield/tests/test_field.py

```diff
@@ -368,19 +366,13 @@ def test_norm(self):
         mesh = df.Mesh(p1=(0, 0, 0), p2=(10, 10, 10), cell=(5, 5, 5))
         f = df.Field(mesh, nvdim=3, value=(2, 2, 2))
 
-        assert np.all(f.norm.value == 2 * np.sqrt(3))
         assert np.all(f.norm.array == 2 * np.sqrt(3))
```

Comment:
Could we use f.norm.update_field_values here?

Reply:
Unfortunately it does not work. If we want this to work I think it would be worth having a chat about ideas on how to implement this.

Reply:
I would suggest we talk and then address this separately.